include/vkgcDefs.h

```diff
@@ -266,6 +267,7 @@ struct PipelineOptions {
 
   ShadowDescriptorTableUsage shadowDescriptorTableUsage; ///< Controls shadow descriptor table.
   unsigned shadowDescriptorTablePtrHigh;                 ///< Sets high part of VA ptr for shadow descriptor table.
+  bool robustAccessEx;                                   ///< If set, support VK_EXT_robustness2
```

Comment:
Maybe add more comments to give a brief description of VK_EXT_robustness2, like this:

///< If set, support VK_EXT_robustness2 (check null descriptor, out-of-range access clamping, ...)

Reply:
I add the comment used for robustAccess in Pipeline.h for a descriptor of VK_EXT_robustness2, is it ok?

Reply:
Oh, it is OK.